PCbuild/pythoncore.vcxproj.filters

```diff
@@ -669,6 +669,9 @@
     <ClInclude Include="..\Include\internal\pycore_intrinsics.h">
       <Filter>Include\cpython</Filter>
     </ClInclude>
+    <ClInclude Include="..\Include\internal\pycore_jit.h">
+      <Filter>Include\internal</Filter>
+    </ClInclude>
```

Comment:
Adding down below with the `<Filter>Python</Filter>`.

Can you clarify what this is for? I see lots of entries that seem to not follow the general pattern of using the parent directory as the filter (`Python\hashtable.c` has the `Modules` filter, `Python\instrumentation.c` has `Source Files`, `Modules\_winapi.c` has `PC`, some headers in `Include\cpython` are filtered as `Include`, some in `Include\internal` are filtered as `Include\cpython`, etc.). Are these deliberate, or just copy-paste errors that have piled up?

Reply:
When you open a C++ project in Visual Studio, it doesn't automatically structure files according to their folders (probably for some historical reason). Instead, it uses this filters file to put them into subfolders. I guess C/C++ doesn't care about folder hierarchy, so it made sense to be able to freely change the view without modifying the source tree.

So yeah, it's mainly just copy-paste errors over time. I don't personally work in VS most of the time, so I'm not sure how bad it is, but just because another window is broken doesn't mean we should break this one.

Reply:
Interesting. Might make sense to generate files like this at some point to reduce errors and busywork.

Reply:
Periodically opening in VS, dragging files into the right place and hitting save should be enough.